### PR TITLE
added missing "position" argument (e_import)

### LINKED ISSUE #2036: Missing "position" argument importing events through pi-manage
### Top-level intent

Importing events does not load position param.

### Steps to reproduce

1. Go to events panel, and create one event with "pre" position. 
2. Export event: `pi-manager event e_export > /tmp/events.py`
3. Import event: `pi-manager event e_import --cleanup /tmp/events.py`
4. Go to events and see the event with "post" position set.

### Expected outcome

`e_import` command should load "position" argument from file

### Actual outcome

`e_import` command always set events position as "post"

### Configuration

* **privacyIDEA version**: master
* **Installation method**: github
* **Python version**: 3.7
* **Operating system**: Ubuntu 18.04
* **Webserver**: development server
* **Token database**: MSSQL (https://github.com/privacyidea/docker/pull/4)

### PR DESCRIPTION
**Pull request by brunocascio**
Ref #2036

Closes #2036

### CODE REVIEW

**Comment by plettich** [comment]
Thanks for the PR @brunocascio
Could You open an issue with some description so we can track the development?

**Comment by brunocascio** [comment]
@plettich done :)

File: pi-manage [hunk 1]
```diff
@@ -1033,7 +1033,8 @@ def e_import(filename, cleanup=False):
                       conditions=event.get("conditions"),
                       ordering=event.get("ordering"),
                       options=event.get("options"),
-                      active=event.get("active"))
+                      active=event.get("active"),
+                      position=event.get("position"))
```
Comment (cornelinux) [inline pi-manage]: If we have an old exported event definition, that does not know post and pre positions, maybe we should default to "post":
~~~~python
   position=event.get("position", "post"))
~~~~

Please consider enhancing this.

Comment (brunocascio) [reply pi-manage]: Hi @cornelinux!

Should "Post" be the default value? If an exported configuration doesn't know about position, assuming "post" could be wrong. Anyway, if yes, I'll do it.

Comment (cornelinux) [reply pi-manage]: If it does not know about positions, the export is from a time, when *all* event handlers where post. (No other positions have been available) 
So post would always be right in this case.

Comment (brunocascio) [reply pi-manage]: Thanks for the explanation!

I have just pushed the change

**Pull request merged**